docfx-extensions/Intent.DocFx.TagExtensions/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
I suspect this can be deleted if you change the `.csproj` to an SDK style file.

Reply:
I ran the upgrade assistant of `dotnet` and it still retained this file exactly as it was. Not sure what the concern here is. The `Reflection` using directive still seems to be needed according to VS.

Reply:
Sorry, I should have specified that I meant this whole _file_ could probably be deleted once the `.csproj` is converted to be SDK style.

I believe that SDK style projects implicitly use the same default values that are in here, so this file is just redundant.